auth_token/utils.py

```diff
@@ -46,9 +51,13 @@ def login(request, user, expiration=True, auth_slug=None, related_objs=None, bac
                                  expiration=expiration, auth_slug=auth_slug, ip=get_ip(request),
                                  backend=backend, allowed_cookie=allowed_cookie, allowed_header=allowed_header)
 
+    if not two_factor_login:
+        token.is_authenticated = True
+        token.save()
+
     for related_obj in related_objs:
         token.related_objects.create(content_object=related_obj)
-    if hasattr(request, 'user'):
+    if hasattr(request, 'user') and not two_factor_login:
```

Comment:
token.is_authenticated

Reply:
It is actually the same as token.is_authenticated in login function.

Reply:
But I think it will be more readable